main/libmemcached/APKBUILD

```diff
@@ -1,49 +1,43 @@
 # Maintainer: Natanael Copa <ncopa@alpinelinux.org>
 pkgname=libmemcached
 pkgver=1.0.18
-pkgrel=1
+pkgrel=2
 pkgdesc="Client library and command line tools for memcached server"
 url="http://libmemcached.org/"
 arch="all"
 license="BSD"
+options="!check" # tests are broken
 depends=""
-depends_dev=""
-makedepends="$depends_dev cyrus-sasl-dev"
+depends_dev="cyrus-sasl-dev"
+makedepends="$depends_dev memcached-dev"
 install=""
 subpackages="$pkgname-dev $pkgname-doc $pkgname-libs"
-source="https://launchpad.net/libmemcached/${pkgver%.*}/${pkgver}/+download/libmemcached-${pkgver}.tar.gz
-	musl-fixes.patch
-	"
+source="https://launchpad.net/${pkgname}/${pkgver%.*}/${pkgver}/+download/${pkgname}-${pkgver}.tar.gz
+	musl-fixes.patch"
 
-_builddir="$srcdir"/libmemcached-$pkgver
-prepare() {
-	local i
-	cd "$_builddir"
-	for i in $source; do
-		case $i in
-		*.patch) msg $i; patch -p1 -i "$srcdir"/$i || return 1;;
-		esac
-	done
-}
+builddir="$srcdir/$pkgname-$pkgver"
 
 build() {
-	cd "$_builddir"
+	cd "$builddir"
 	./configure \
-		--build=$CBUILD \
-		--host=$CHOST \
+		--build="$CBUILD" \
```

Comment:
why the quoting for `--build` and `--host`?

Reply:
These should be quoted all the time.
The only case where the current usage isn't a problem is when `CBUILD` and `CHOST` are unmodified by word splitting.

Reply:
I agree that it’s better to always quote all variables defined outside of the APKBUILD itself (i.e. author can’t be sure that it will not contain any spaces).